packages/ember-testing/lib/helpers/click.js

```diff
@@ -1,3 +1,18 @@
+/**
```

Comment:
can you move this comment block down to directly above the `click` function below?